dag/tourism.yml

```diff
@@ -28,3 +28,15 @@ steps:
     - data://garden/regions/2023-01-01/regions
   data://grapher/oecd/2024-08-21/co2_air_transport:
     - data://garden/oecd/2024-08-21/co2_air_transport
+  data://meadow/tourism/2024-08-17/unwto_environment:
```

Comment:
I would separate these blocks by data set and add a comment, so you can find them quickly when e.g. updating these data sets